communication/src/main/java/com/appcoins/communication/requester/MessageRequesterSynchronizer.java

```diff
@@ -0,0 +1,13 @@
+package com.appcoins.communication.requester;
+
+import android.os.Parcelable;
+
+class MessageRequesterSynchronizer {
+  public MessageRequesterSynchronizer() {
+    StaticMessageResponseSynchronizer.init();
+  }
+
+  public Parcelable waitMessage(long requestCode) throws InterruptedException {
+    return StaticMessageResponseSynchronizer.waitMessage(requestCode);
+  }
+}
```

Comment:
What's the advantage of using a class that all it does it call a static method? Shouldn't the class that calls this method, call the static method directly?

Reply:
I created this class to isolate as much as i can the static code so i can test the rest of the code.